### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by warlice**

### CODE REVIEW

File: app/src/main/java/com/boringdroid/systemui/utils/ParseUtils.java [hunk 1]
```diff
@@ -101,28 +104,8 @@ public static void parseGpsData(Context context) {
     }
 
     public static void parseListXML(Context context) {
-//        try {
         InputStream inputStream = context.getResources().openRawResource(R.raw.comp_config);
         parseList(context, inputStream);
-//            BufferedReader bufferedReader = new BufferedReader(new InputStreamReader(inputStream));
```
Comment (warlice) [inline app/src/main/java/com/boringdroid/systemui/utils/ParseUtils.java:107]: remove commented code

File: app/src/main/java/com/boringdroid/systemui/view/SystemStateLayout.kt [hunk 2]
```diff
@@ -200,9 +224,17 @@ class SystemStateLayout(context: Context?, attrs: AttributeSet?) :
 //        intent.component = cn;
 //        intent.flags = Intent.FLAG_ACTIVITY_NEW_TASK
 //        context.startActivity(intent)
-
         netCenterWindow?.ifShowNetCenterView()
         listener?.syncVisible(Utils.WIFIWINDOW_VISIBLE)
+
+
+        GlobalScope.launch(Dispatchers.Main) {
+            // 在IO线程上执行异步操作
```
Comment (warlice) [inline app/src/main/java/com/boringdroid/systemui/view/SystemStateLayout.kt]: comment in english

File: app/src/main/java/com/boringdroid/systemui/view/SystemStateLayout.kt [hunk 3]
```diff
@@ -200,9 +224,17 @@ class SystemStateLayout(context: Context?, attrs: AttributeSet?) :
...
+            val result = withContext(Dispatchers.IO) {
+                // 模拟长时间的网络请求或其他耗时操作
```
Comment (warlice) [inline app/src/main/java/com/boringdroid/systemui/view/SystemStateLayout.kt]: comment in english

**Comment by xudingqiang** [comment]
fix conflict

**Pull request merged**